config/application.rb

```diff
@@ -34,6 +34,17 @@ class Application < Rails::Application
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
 
+    # Use cookies for sessions
+    config.session_store :cookie_store, key: '_keygen_session',
```

Comment:
Should we use a `Session` model instead? It'd give us auditable sessions and also make them revocable.

Reply:
We could start with a `UserSession` model.

Reply:
Ref https://github.com/rails/rails/pull/52328.

Reply:
Maybe we should do the session stuff in a separate PR?